core/eolearn/core/utilities.py

```diff
@@ -7,6 +7,7 @@
 from collections import OrderedDict
```

Comment:
There are quite a few pylint issues in module `eolearn.core.utilities`.

Issues: https://travis-ci.org/sentinel-hub/eo-learn/jobs/568268544#L1274